src/tests/QueryContext.test.js

```diff
@@ -0,0 +1,11 @@
+import Search from '../components/Search';
+import { render, screen} from '@testing-library/react';
+
+describe('Query context', () => {
+    test('initial value is empty', () => {
+        render(<Search/>);
+        const inputText = screen.getByTestId("search-input");
+        expect(inputText.textContent).toBe("");
```

Comment:
Something like this should be better.
```suggestion
         const value = { query: 'test' };
        render(<QueryContext.Provider value={value}><Search/></QueryContext.Provider>);
        const inputText = screen.getByTestId("search-input");
        expect(inputText.value).toBe("test");
```

Reply:
I changed the test to retrieve the initial value should be Wizeline but was unable to test it this way because it says that QueryContext.provider is undefined.